src/explore-education-statistics-common/src/components/__tests__/ContentHtml.test.tsx

```diff
@@ -248,16 +248,25 @@ describe('ContentHtml', () => {
       ).toHaveAttribute('href', 'https://gov.uk/TEST');
     });
 
-    test('opens external links in a new tab', () => {
-      render(
-        <ContentHtml html='<a href="https://gov.uk/">External link</a>' />,
-      );
+    test.each([
+      ['https://gov.uk/'],
+      [
+        'https://admin.explore-education-statistics.service.gov.uk/publication/abc/release/123/content',
+      ],
+      ['https://www.dev.explore-education-statistics.service.gov.uk/'],
+    ])('opens external links in a new tab', (target: string) => {
+      render(<ContentHtml html={`<a href="${target}">External link</a>`} />);
+
       const link = screen.getByRole('link', {
         name: 'External link (opens in a new tab)',
       });
-      expect(link).toHaveAttribute('href', 'https://gov.uk/');
+      expect(link).toHaveAttribute('href', target);
       expect(link).toHaveAttribute('target', '_blank');
-      expect(link).toHaveAttribute('rel', 'noopener noreferrer');
+
+      const rel = link.getAttribute('rel');
+      expect(rel).toContain('noopener');
+      expect(rel).toContain('noreferrer');
+      expect(rel).toContain('nofollow');
```

Comment:
I'd rather not revert back to `toHaveAttribute` because I want to be order agnostic (i.e. the rel could be `noopener noreferrer` or `noreferrer noopener` and the test should still pass). 

We could maybe do `expect(rel.split(' ')).toHaveLength(3)` to catch the issues you raise?